templates/api_server_deployment.yml

```diff
@@ -39,8 +39,10 @@ spec:
           imagePullPolicy: Always
           resources:
             requests:
+              cpu: 500m
               memory: 300Mi
             limits:
+              cpu: 1000m
               memory: 1.2Gi
```

Comment:
I know you didn't change this, but for @cloudfoundry/cf-capi how did we choose this memory limit? 

On BOSH the limit ~4GB:
https://github.com/cloudfoundry/capi-release/blob/6f8899976561e64eab8c9804b1ce772083bbf68c/jobs/cloud_controller_ng/spec#L878-L886

In production envs I've seen a well-used CF API settle in at around 2.5-3GB (occasionally bursting higher for certain requests), so I think we should make it match the capi-release limits.

Reply:
It was probably a raw multiple of the initially-deployed memory usage. We set these to keep the API from getting OOM killed, not based on any experiment to produce memory bloat or "realistic" workloads.

Reply:
That's good context, thanks @cwlbraa.

I wonder if we should just make the very forgiving for now (either meet or exceed the BOSH defaults). The main use for them is to just be a good neighbor and avoid starving other `Pods` of resources, but I feel like if we're targeting a "developer edition" 1.0 they might be more frustrating than helpful if they're too strict.